Wired.js

```diff
@@ -0,0 +1,300 @@
+			}
+			if (item.tags) { // catch volume/issue if in tags
+				var match = null;
+				for (var tagCount in item.tags) {
+					match = item.tags[tagCount].match(/^(\d{2})\.(\d{2})$/);
+					if (match) {
+						item.volume = match[1];
+						item.issue = parseInt(match[2]);
+						item.tags.splice(tagCount,1);
+						break;
+					}
+				}
+			}
+		}
+		item.complete();
+	});
+	translator.getTranslatorObject(function(trans) {
+		trans.doWeb(doc, url);
+	});
+}
+
+
+function getSearchResults(doc, checkOnly) {
+	var items = {};
+	var found = false;
+	var rows = doc.querySelectorAll('div.card-component h2, li.archive-item-component h2, section.c-card-section article.c-card h3');
+	var links = doc.querySelectorAll('.card-component__description > a:first-of-type, li.archive-item-component > a:first-of-type, section.c-card-section article.c-card > a');
+	for (let i=0; i<rows.length; i++) {
+		let href = links[i].href;
+		let title = ZU.trimInternal(rows[i].textContent);
+		if (!href || !title) continue;
+		if (checkOnly) return true;
+		found = true;
+		items[href] = title;
+	}
+	return found ? items : false;
+}
+
+
+function doWeb(doc, url) {
+	switch (detectWeb(doc, url)) {
+		case "multiple":
+			Zotero.selectItems(getSearchResults(doc, false), function (items) {
+				if (!items) {
+					return true;
+				}
+				var articles = [];
+				for (var i in items) {
+					articles.push(i);
+				}
+				ZU.processDocuments(articles, scrape);
+			});
+			break;
+		case "magazineArticle":
+			scrape(doc, url);
+			break;
+	}
+}
+/** BEGIN TEST CASES **/
+var testCases = [
+	{
+		"type": "web",
+		"url": "https://www.wired.com/2011/03/ff_kickstarter/",
+		"items": [
+			{
+				"itemType": "magazineArticle",
+				"title": "How Kickstarter Became a Lab for Daring Prototypes and Ingenious Products",
+				"creators": [
+					{
+						"firstName": "Carlye",
+						"lastName": "Adler",
+						"creatorType": "author"
+					}
+				],
+				"date": "2011-03-18",
+				"ISSN": "1059-1028",
+				"abstractNote": "A group of modern makers kick-started a website for passion projects and patrons.",
+				"issue": 4,
+				"language": "en-US",
+				"libraryCatalog": "www.wired.com",
+				"publicationTitle": "Wired",
+				"url": "https://www.wired.com/2011/03/ff_kickstarter/",
+				"volume": "19",
+				"attachments": [
+					{
+						"title": "Snapshot"
+					}
+				],
+				"tags": [
+					{
+						"tag": "Kickstarter"
+					},
+					{
+						"tag": "Startups"
+					},
+					{
+						"tag": "features"
+					}
+				],
+				"notes": [],
+				"seeAlso": []
+			}
+		]
+	},
+	{
+		"type": "web",
+		"url": "https://www.wired.com/story/in-defense-of-the-vegan-hot-dog/",
+		"items": [
+			{
+				"itemType": "magazineArticle",
+				"title": "In Defense of the Vegan Hot Dog",
+				"creators": [
+					{
+						"firstName": "Emily",
+						"lastName": "Dreyfuss",
+						"creatorType": "author"
+					}
+				],
+				"date": "2018-07-04T14:00:00.000Z",
```

Comment:
@zuphilip do we have a position on including or removing the timestamp from dates? I don't really care either way, but we should unify.

Reply:
That should be fine and we already have the timestamps in various other translators, cf.
```
$ grep -P "\d\d\d\d-\d\d-\d\dT\d\d:" *.js
AllAfrica.js:                           "date": "2011-10-18T02:24:01+0000",
AllAfrica.js:                           "date": "2011-10-04T09:49:38+0000",
BBC Genome.js:                          "date": "1933-10-28T20:00",
BBC Genome.js:                          "date": "1969-07-21T06:00",
Bloomberg.js:                           "date": "2012-01-04T00:00:00.001Z",
Bloomberg.js:                           "date": "2012-01-05T00:01:34.000Z",
Boston Review.js:                               "date": "2012-09-01T00:00:00-04:00",
Boston Review.js:                               "date": "2013-05-21T00:00:00-04:00",
Boston Review.js:                               "date": "2013-06-07T00:00:00-04:00",
Christian Science Monitor.js:                           "date": "2011-12-04T09:45:36-05:00",
Christian Science Monitor.js:                           "date": "2011-12-02T11:00:00-05:00",
Denik CZ.js:                            "date": "2017-01-23T08:20:00+01:00",
Denik CZ.js:                            "date": "2017-12-31T10:08:00+01:00",
Denik CZ.js:                            "date": "2018-01-01T14:22:00+01:00",
Denik CZ.js:                            "date": "2018-01-02T00:30:00+01:00",
Embedded Metadata.js:                           "date": "2013-12-22T11:58:34+00:00",
Embedded Metadata.js:                           "date": "2015-08-05T12:05:17Z",
Embedded Metadata.js:                           "date": "2016-01-07T08:20:02-05:00",
Epicurious.js:                          "date": "2010-10-05T04:00:00.000Z",
FAZ.NET.js:                             "date": "2011-06-13T06:00:00+0200",
FAZ.NET.js:                             "date": "2017-08-17T11:56:48+0200",
Foreign Affairs.js:                             "date": "2009-01-28T19:05:53-05:00",
Github.js:                              "date": "2018-04-14T04:06:44Z",
Github.js:                              "extra": "original-date: 2011-10-27T07:46:48Z",
Github.js:                              "date": "2018-03-22T14:50:46Z",
Github.js:                              "extra": "original-date: 2011-04-13T07:08:41Z",
Github.js:                              "date": "2018-04-10T01:48:27Z",
Github.js:                              "extra": "original-date: 2015-05-19T09:24:38Z",
Github.js:                              "date": "2018-04-02T21:32:18Z",
Github.js:                              "extra": "original-date: 2012-05-20T07:53:58Z",
Google Presentation.js:                         "date": "2017-05-23T06:24:16.216Z",
Google Presentation.js:                         "date": "2012-11-20T19:32:53.370Z",
Huff Post.js:                           "date": "2011-11-30T00:53:25Z",
Huff Post.js:                           "date": "2011-11-29T21:53:11Z",
Huff Post.js:                           "date": "2016-10-31T18:48:44Z",
La Croix.js:                            "date": "2016-05-27T17:31:45+02:00",
La Croix.js:                            "date": "2016-01-19T10:08:37+01:00",
La Croix.js:                            "date": "2016-05-29T17:32:54+02:00",
LA Times.js:                            "date": "2014-05-31T16:05:00PDT",
Le Figaro.js:                           "date": "2016-05-27T16:12:21+02:00",
Le Figaro.js:                           "date": "2016-02-19T15:52:31+01:00",
Le Figaro.js:                           "date": "2016-05-28T19:08:03+02:00",
Le Monde.js:                            "date": "2015-03-13T10:37:46+01:00",
Le Monde.js:                            "date": "2015-03-13T11:50:19+01:00",
Le Monde.js:                            "date": "2015-03-13T22:03:36+01:00",
Le Monde.js:                            "date": "2013-09-28T09:26:28+02:00",
Medium.js:                              "date": "2014-04-22T16:02:32.005Z",
New Zealand Herald.js:                          "date": "2017-07-18T02:45:22Z",
New Zealand Herald.js:                          "date": "2017-07-19T18:00:00Z",
New Zealand Herald.js:                          "date": "2017-07-21T17:00:00Z",
New Zealand Herald.js:                          "date": "2017-07-21T17:00:00Z",
NewsnetTamedia.js:                              "date": "2018-04-17T18:33:00",
NZZ.ch.js:                              "date": "2016-05-31T07:45:25.872Z",
NZZ.ch.js:                              "date": "2016-05-28T23:00:00.000Z",
Paris Review.js:                                "date": "2011-11-07T13:15:51-05:00",
Roll Call.js:                           "date": "2012-11-09T11:08:02Z",
Roll Call.js:                           "date": "2013-05-12T02:49:35Z",
Spiegel Online.js:              date = date.replace(/T.+/,""); // e.g. date = "2014-10-20T12:10:00+0200"
Store norske leksikon.js:                               "date": "2014-08-27T09:47:59.567+02:00",
Store norske leksikon.js:                               "date": "2016-02-23T13:23:40.665+01:00",
Store norske leksikon.js:                               "date": "2014-09-28T14:36:15.097+02:00",
Store norske leksikon.js:                               "date": "2017-02-20T14:12:53.744+01:00",
Store norske leksikon.js:                               "date": "2014-09-28T16:09:18.163+02:00",
Store norske leksikon.js:                               "date": "2017-02-28T12:16:33.244+01:00",
Tagesspiegel.js:                                "date": "2012-10-04T22:00:00Z",
Tagesspiegel.js:                                "date": "2017-06-27T16:20:46Z",
taz.de.js:                              "date": "2011-03-23T18:22:00+02:00",
taz.de.js:                              "date": "2017-06-28T00:00:00+01:00",
The Economist.js:                               "date": "2011-11-12T00:00:00+0000",
The Economist.js:                               "date": "2017-08-19T00:00:00Z",
The Economist.js:                               "date": "2017-11-15T07:47:30Z",
The Guardian.js:                                "date": "2013-03-06T15:38:00.000Z",
The Guardian.js:                                "date": "2013-03-06T20:04:00.000Z",
The Guardian.js:                                "date": "2013-02-26T17:18:55.000Z",
The Guardian.js:                                "date": "2015-02-18T20:22:30.000Z",
The Guardian.js:                                "date": "2011-11-27T00:05:08.000Z",
The Hamilton Spectator.js:                              "date": "2011-11-16T21:54:00Z",
The Hindu.js:                           "date": "2013-07-14T17:42:52+05:30",
The Hindu.js:                           "date": "2013-07-14T18:29:36+05:30",
The Hindu.js:                           "date": "2013-07-14T17:57:17+05:30",
The Hindu.js:                           "date": "2013-07-14T13:57:47+05:30",
The Intercept.js:                               "date": "2017-06-05T19:44:23+00:00",
The Intercept.js:                               "date": "2015-11-19T18:04:35+00:00",
The Nation.js:                          "date": "2011-11-16T23:16:53-04:00",
The Nation.js:                          "date": "2016-12-12T10:00:44-04:00",
The New Yorker.js:                              "date": "2011-10-20T04:00:00.000Z",
The New Yorker.js:                              "date": "2012-02-24T23:12:35.000Z",
The New Yorker.js:                              "date": "2017-06-12T04:00:00Z",
TheMarker.js:                           "date": "2015-08-15T13:23:18+0300",
Twitter.js:                             "date": "2011-08-22T04:52",
Twitter.js:                             "date": "2018-01-31T12:00",
Vanity Fair.js:                         "date": "2011-08-10T00:00:00.000-04:00",
Vanity Fair.js:                         "date": "2011-05-01T00:00:00.000-04:00",
Vanity Fair.js:                         "date": "2011-08-10T00:00:00.000-04:00",
Vimeo.js:                               "date": "2011-10-26T22:29:03-04:00",
Wall Street Journal.js:                         "date": "2011-11-19T05:01:00.000Z",
Wall Street Journal.js:                         "date": "2012-01-07T05:01:00.000Z",
Wall Street Journal.js:                         "date": "2012-01-06T21:22:12-05:00",
Wall Street Journal.js:                         "date": "2012-01-07T10:00:55-05:00",
Wall Street Journal.js:                         "date": "2014-09-07T01:37:00.000Z",
Washington Monthly.js:                          "date": "2011-04-26T16:49:38+00:00",
Washington Monthly.js:                          "date": "2011-11-19T12:55:31+00:00",
washingtonpost.com.js:                          "date": "2011-09-30T08:06-500",
washingtonpost.com.js:                          "date": "2011-11-07T09:49-500",
washingtonpost.com.js:                          "date": "1991-04-07T12:00-500",
Welt Online.js:                         "date": "2011-03-26T06:50:04Z",
Wikidata QuickStatements.js:            //e.g. +1967-01-17T00:00:00Z/11
Wikidata.js:                            "date": "2002-04-01T00:00:00Z",
Wikidata.js:                            "date": "2013-06-01T00:00:00Z",
Wikidata.js:                            "date": "1974-08-30T00:00:00Z",
Wikidata.js:                            "date": "1963-01-01T00:00:00Z",
Wikidata.js:                            "date": "1999-01-01T00:00:00Z",
Wikipedia.js:                           "date": "2012-04-06T20:11:32Z",
Wikipedia.js:                           "date": "2012-04-03T14:41:27Z",
Wikipedia.js:                           "date": "2016-10-24T18:43:24Z",
XML ContextObject.js:           "input": "<ctx:context-objects xmlns:xsi='http://www.w3.org/2001/XMLSchema-instance' xsi:schemaLocation='info:ofi/fmt:xml:xsd:ctx http://www.openurl.info/registry/docs/info:ofi/fmt:xml:xsd:ctx' xmlns:ctx='info:ofi/fmt:xml:xsd:ctx'>\n<ctx:context-object timestamp='2010-01-02T16:55:48-05:00' encoding='info:ofi/enc:UTF-8' version='Z39.88-2004' identifier=''>\n <ctx:referent>\n  <ctx:metadata-by-val>\n   <ctx:format>info:ofi/fmt:xml:xsd:journal</ctx:format>\n   <ctx:metadata>\n    <journal xmlns:rft='info:ofi/fmt:xml:xsd:journal' xsi:schemaLocation='info:ofi/fmt:xml:xsd:journal http://www.openurl.info/registry/docs/info:ofi/fmt:xml:xsd:journal'>\n\t <rft:atitle>Acute Myocardial Infarction in the Medicare population: process of care and clinical outcomes</rft:atitle>\n\t <rft:spage>2530</rft:spage>\n\t <rft:date>1992</rft:date>\n\t <rft:stitle>Journal of the American Medical Association</rft:stitle>\n\t <rft:genre>article</rft:genre>\n\t <rft:volume>18</rft:volume>\n\t <rft:epage>2536</rft:epage>\n\t <rft:au>I S Udvarhelyi</rft:au>\n\t <rft:au>C A Gatsonis</rft:au>\n\t <rft:au>A M Epstein</rft:au>\n\t <rft:au>C L Pashos</rft:au>\n\t <rft:au>J P Newhouse</rft:au>\n\t <rft:au>B J McNeil</rft:au>\n\t</journal>\n   </ctx:metadata>\n  </ctx:metadata-by-val>\n </ctx:referent>\n</ctx:context-object>\n</ctx:context-objects>",
XML ContextObject.js:           "input": "<ctx:context-objects xmlns:xsi='http://www.w3.org/2001/XMLSchema-instance' xsi:schemaLocation='info:ofi/fmt:xml:xsd:ctx http://www.openurl.info/registry/docs/info:ofi/fmt:xml:xsd:ctx' xmlns:ctx='info:ofi/fmt:xml:xsd:ctx'> <ctx:context-object timestamp='2015-03-16T15:37:18-04:00' encoding='info:ofi/enc:UTF-8' version='Z39.88-2004' identifier=''><ctx:referent><ctx:metadata-by-val><ctx:format>info:ofi/fmt:xml:xsd:journal</ctx:format><ctx:metadata><journal xmlns:rft='info:ofi/fmt:xml:xsd:journal' xsi:schemaLocation='info:ofi/fmt:xml:xsd:journal http://www.openurl.info/registry/docs/info:ofi/fmt:xml:xsd:journal'><rft:quarter>4356</rft:quarter><rft:atitle>Molecular structure of nucleic acids; a structure for deoxyribose nucleic acid</rft:atitle><rft:spage>737</rft:spage><rft:date>1953</rft:date><rft:jtitle>Nature</rft:jtitle><rft:genre>article</rft:genre><rft:volume>171</rft:volume><rft:epage>738</rft:epage><rft:au>J D Watson</rft:au><rft:au>F H C Crick</rft:au></journal></ctx:metadata></ctx:metadata-by-val></ctx:referent></ctx:context-object>  <ctx:context-object timestamp='2015-03-16T15:37:18-04:00' encoding='info:ofi/enc:UTF-8' version='Z39.88-2004' identifier=''><ctx:referent><ctx:metadata-by-val><ctx:format>info:ofi/fmt:xml:xsd:journal</ctx:format><ctx:metadata><journal xmlns:rft='info:ofi/fmt:xml:xsd:journal' xsi:schemaLocation='info:ofi/fmt:xml:xsd:journal http://www.openurl.info/registry/docs/info:ofi/fmt:xml:xsd:journal'><rft:quarter>4</rft:quarter><rft:atitle>On the electrodynamics of moving bodies</rft:atitle><rft:spage>1</rft:spage><rft:date>1905</rft:date><rft:stitle>Annalen Der Physik</rft:stitle><rft:genre>article</rft:genre><rft:volume>17</rft:volume><rft:epage>26</rft:epage><rft:au>A Einstein</rft:au></journal></ctx:metadata></ctx:metadata-by-val></ctx:referent></ctx:context-object> </ctx:context-objects>",
Zenodo.js:                              "accessDate": "2018-03-07T13:45:35Z",

```